airflow/executors/base_executor.py

```diff
@@ -540,7 +540,9 @@ def terminate(self):
         """Get called when the daemon receives a SIGTERM."""
         raise NotImplementedError
 
-    def cleanup_stuck_queued_tasks(self, tis: list[TaskInstance]) -> list[str]:  # pragma: no cover
+    def cleanup_stuck_queued_tasks(
+        self, tis: list[TaskInstance]
+    ) -> Iterable[TaskInstance]:  # pragma: no cover
         """
         Handle remnants of tasks that were failed because they were stuck in queued.
```

Comment:
We should reword this docsrting too.

Reply:
no longer changing this function's behavior